tests/link/numba/signal/test_conv.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+import pytest
+
+from pytensor.tensor import dmatrix
+from pytensor.tensor.signal import convolve1d
+from tests.link.numba.test_basic import compare_numba_and_py
+
+
+pytestmark = pytest.mark.filterwarnings("error")
+
+
+@pytest.mark.parametrize("mode", ["full", "valid", "same"])
+def test_convolve1d(mode):
+    x = dmatrix("x")
+    y = dmatrix("y")
+    out = convolve1d(x[None], y[:, None], mode=mode)
+
+    rng = np.random.default_rng()
+    test_x = rng.normal(size=(3, 5))
+    test_y = rng.normal(size=(7, 11))
+    # Object mode is not supported for numba
```

Comment:
Why not?

Reply:
Specifically it's the numba dispatch of Blockwise that doesn't have an obj implementation.. I'll tweak the comment

Reply:
Clarified comment